data/tasks.txt

```diff
@@ -0,0 +1,2 @@
+T | 1 | call mom
```

Comment:
As data/tasks.txt is going to be constantly changing, might be good to not track it. Consider adding it to gitignore as well as removing this cached version to prevent any further tracking by git.